src/components/PricePlan.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from 'styled-components';
+import { PlanType, PriceType } from '../types';
+import icon from '../assets/icon-music.svg';
+
+interface Props {
+  plan: PlanType;
+  selected: boolean;
+  onHandleClick: (val: string) => void;
+}
+
+type ButtonProps = {
+  onClick: (event: React.MouseEventHandler<HTMLButtonElement>) => void;
+};
+
+const PricePlan = ({ plan, selected, onHandleClick }: Props) => {
+  return (
+    <Container>
+      <img src={icon} alt="Music Note" />
+      <PriceContainer>
+        <Heading>{plan.type} Plan</Heading>
+        <Text>${plan.amount}/year</Text>
+      </PriceContainer>
+      {!selected && (
+        <ActionLink onClick={() => onHandleClick(plan.id)}>Change</ActionLink>
```

Comment:
Nit: Change text is hard coded here. Might be a good a idea to allow the user to pass it through?

Reply:
I've added a prop that is passed down from the SummaryCard component through the PricePlanList to the PricePlan component